### ISSUE TITLE
Return A11y score as text? 

### ISSUE THREAD

**Issue by jameschetwood** [omgovich/colord#107]
It would be great if this library could return the colour contrast score in text eg how this other library works:

https://github.com/tmcw/wcag-contrast#score
`score(10); // = 'AAA'`

Currently you would need to use the `isReadable` function to return booleans and manually workout the text score, right? 

```
const aaa = colord("#e60000").isReadable("#ffff47", { level: "AAA" }); // false
const aaaLarge = colord("#e60000").isReadable("#ffff47", { level: "AAA", size: "large" }); // true 

if(aaa) return 'AAA'
if(aaaLarge) return 'AAA large'
return 'FAIL'
```

**Comment by dkryaklin** [comment]
Added `readableScore()` in [@colordx/core](https://github.com/dkryaklin/colordx)

```ts
import { colordx, extend } from '@colordx/core';
import a11y from '@colordx/core/plugins/a11y';

extend([a11y]);

colordx('#000000').readableScore('#ffffff') // 'AAA'
colordx('#e60000').readableScore('#ffff47') // 'AA'
colordx('#949494').readableScore('#ffffff') // 'AA large'
colordx('#aaaaaa').readableScore('#ffffff') // 'fail'
```

Returns `'AAA' | 'AA' | 'AA large' | 'fail'` based on the highest WCAG level the contrast ratio satisfies
